rules/windows/privilege_escalation_krbrelayup_suspicious_logon.toml

```diff
@@ -0,0 +1,74 @@
+[metadata]
+creation_date = "2022/04/26"
+maturity = "production"
+updated_date = "2022/04/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a suspicious local successful logon event where the Logon Package is kerberos, the remote address is set to
+localhost and the target user SID is the builtin local Administrator account, this may indicate an attempt to leverage a
+Kerberos relay attack variant that can be used to elevate privilege locally from filtered administrator token to a token
+with full System privileges.
```

Comment:
```suggestion
Identifies a suspicious local successful logon event where the Logon Package is Kerberos, the remote address is set to
localhost, and the target user SID is the built-in local Administrator account. This may indicate an attempt to leverage
a Kerberos relay attack variant that can be used to elevate privilege locally from a filtered administrator token to a
token with full System privileges.
```